crates/bevy_ecs/src/reflect/entity_commands.rs

```diff
@@ -103,36 +123,43 @@ pub trait ReflectCommandExt {
     /// # Example
     ///
     /// ```
-    /// // Note that you need to register the component type in the AppTypeRegistry prior to using
+    /// // Note that you need to register the component/bundle type in the AppTypeRegistry prior to using
     /// // reflection. You can use the helpers on the App with `app.register_type::<ComponentA>()`
-    /// // or write to the TypeRegistry directly to register all your components
+    /// // or write to the TypeRegistry directly to register all your components and bundles
     ///
     /// # use bevy_ecs::prelude::*;
     /// # use bevy_ecs::reflect::ReflectCommandExt;
     /// # use bevy_reflect::{FromReflect, FromType, Reflect, TypeRegistry};
     ///
-    /// // A resource that can hold any component that implements reflect as a boxed reflect component
+    /// // A resource that can hold any component or bundle that implements reflect as a boxed reflect
     /// #[derive(Resource)]
     /// struct Prefab{
     ///     entity: Entity,
-    ///     component: Box<dyn Reflect>,
+    ///     data: Box<dyn Reflect>,
     /// }
     /// #[derive(Component, Reflect, Default)]
     /// #[reflect(Component)]
     /// struct ComponentA(u32);
     /// #[derive(Component, Reflect, Default)]
     /// #[reflect(Component)]
     /// struct ComponentB(String);
+    /// #[derive(Bundle, Reflect, Default)]
+    /// #[reflect(Bundle)]
+    /// struct BundleA {
+    ///     a: ComponentA,
+    ///     b: ComponentB,
+    /// }
     ///
     /// fn remove_reflect_component(
     ///     mut commands: Commands,
     ///     prefab: Res<Prefab>
     ///     ) {
-    ///     // Prefab can hold any boxed reflect component. In this case either
-    ///     // ComponentA or ComponentB. No matter which component is in the resource though,
-    ///     // we can attempt to remove any component of that same type from an entity.
+    ///     // Prefab can hold any boxed reflect component or bundle. In this case either
+    ///     // ComponentA, ComponentB, or BundleA. No matter which component or bundle is in the resource though,
+    ///     // we can attempt to remove any component (or set of components in the case of a bundle)
+    ///     // of that same type from an entity.
     ///     commands.entity(prefab.entity)
-    ///         .remove_reflect(prefab.component.reflect_type_path().to_owned());
+    ///         .remove_reflect(prefab.data.reflect_type_path().to_owned());
     /// }
     ///
     /// ```
```

Comment:
It would be nice to include safe removal methods and error handling examples.

Reply:
What do you mean by safe removal methods? Would this not be considered safe since it doesn't panic?

Reply:
My English is bad. I meant “safe” to mean preventing unexpected behavior, not preventing panic.

But based on the replies above, I don't think it's necessary. The overall feel of the finished code.